main.js

```diff
@@ -0,0 +1,270 @@
+const main = function() {
+  // body...
+
+  // This data will help with some logic in the game.
+  var data = {
+    board: [
+      ['1', '2','3'],
+      ['4', '5','6'],
+      ['7', '8','9']
+    ],
+    playerWins: false,
+    counter: 0,
+    positionsTaken: ['0']
+  }
+  // Get name of players. Save them in an array
+  const player1Name = prompt("What is player 1 name?");
+  const player2Name = prompt("What is player 2 name?");
+  const players = [player1Name, player2Name];
+  // Array to choose a random character (X or O)
+  let chars = ['X', 'O'];
+  let getPlayer1Char = chars[getRandomInt(chars.length)];
+  getPlayer2Char = '';
+  // Give the opposite character to the user2 depending,
+  // on what user1 got in the random selector.
+  if(getPlayer1Char == 'X') {
+    getPlayer2Char = 'O'
+  } else {
+    getPlayer2Char = 'X'
+  }
+  // Array to choose a random character to player1,
+  // which will go first. Basically getting a random player to go first.
+  let getPlayer1 = players[getRandomInt(chars.length)];
+  let getPlayer2 = '';
+  if(getPlayer1 === player1Name) {
+    getPlayer2 = player2Name;
+  } else {
+    getPlayer2 = player1Name;
+  }
+  // Create the players with their information.
+  const player1 = new Player(getPlayer1, getPlayer1Char);
+  const player2 = new Player(getPlayer2, getPlayer2Char);
+
+  console.log(`Player1 = ${player1.name}, player2 = ${player2.name}`);
+  console.log(`Player1 = ${player1.char}, player2 = ${player2.char}`);
+  // Create a new game with the 2 players information and the data object.
```

Comment:
Overall, there a few spots where newlines would make these comments easier to read